orca-core/src/main/groovy/com/netflix/spinnaker/orca/pipeline/util/PackageInfo.groovy

```diff
@@ -44,11 +44,12 @@ class PackageInfo {
     potentialUrl ==~ /\b(https?|ssh):\/\/.*/
   }
 
-  public Map findTargetPackage() {
+  public Map findTargetPackage(boolean allowMissingPackageInstallation) {
     Map requestMap = [:]
     // copy the context since we may modify it in createAugmentedRequest
     requestMap.putAll(stage.execution.context)
     requestMap.putAll(stage.context)
+    requestMap['allowMissingPackageInstallation'] = allowMissingPackageInstallation
```

Comment:
Any reason not to pass this to `createAugmentedRequest()` as an argument, instead of adding it into the `requestMap`? Might read cleaner that way.